package.json

```diff
@@ -9,10 +9,11 @@
     "prettier": "^1.19.1",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
+    "react-firestore": "^1.0.1",
+    "react-icons": "^3.11.0",
```

Comment:
I see you added a new package! I'm excited to hear you talk about how you found it and why you decided to use it. I've never heard of it before!